packages/@glimmer/node/tests/node-test.ts

```diff
@@ -38,7 +38,13 @@ function commonSetup() {
 function render<T>(template: Template<T>, self: any) {
   let result;
   env.begin();
-  result = template.render(new UpdatableReference(self), root, new TestDynamicScope());
+  let templateIterator = template.render(new UpdatableReference(self), root, new TestDynamicScope());
+
+  do {
+    result = templateIterator.next();
+  } while (!result.done);
+
+  result = result.value;
```

Comment:
this is because result is typed as any but this doesn't seem like a good thing todo.